code/modules/pda/ai_pda.dm

```diff
@@ -2,6 +2,19 @@
 /obj/item/pda/silicon
 	detonate = FALSE
 	ttone = "data"
+	model_name = "Thinktronic 6000 Internal Personal Data Assistant"
+	silicon_pda = TRUE
+	programs = list(
```

Comment:
add a comment saying what is different from the parent here, also remove the // here our games go comment